packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -120,11 +120,25 @@ class AndroidApk extends ApplicationPackage {
     final String packageId = manifests.first.getAttribute('package');
 
     String launchActivity;
-    for (xml.XmlElement category in document.findAllElements('category')) {
-      if (category.getAttribute('android:name') == 'android.intent.category.LAUNCHER') {
-        final xml.XmlElement activity = category.parent.parent;
-        final String enabled = activity.getAttribute('android:enabled');
-        if (enabled == null || enabled == 'true') {
+    for (xml.XmlElement activity in document.findAllElements('activity')) {
+      final String enabled = activity.getAttribute('android:enabled');
+      if(enabled!=null && enabled!='true')
```

Comment:
Can you please add spaces around comparison operators? `enabled!=null` -> `enabled != null`

Also, can you use equality where possible? `enabled!='true'` -> `enabled == 'false'`